crates/aptos/src/account/create.rs

```diff
@@ -38,10 +38,11 @@ pub struct CreateAccount {
     #[clap(flatten)]
     node: NodeOptions,
 
-    /// Public Key of account you want to create
```

Comment:
I worry that without this explanation the inputs will be confusing here. 

For example, the public key is the key for the account to be created. And the private key is the key that is funding the transaction.

Reply:
Yeah, I'm still trying to figure out a better way of doing it, maybe clap allows for overriding the documentation.

Over today I'm going to do some documentation rewrites to make sure everything is clean, and I might change this appropriately.

Reply:
> maybe clap allows for overriding the documentation

Couldn't find a way when I briefly looked